libjlm/include/jlm/backend/hls/rhls2firrtl/mlirgen.hpp

```diff
@@ -45,7 +45,7 @@ namespace jlm {
 			MLIRGenImpl(mlir::MLIRContext &context) : builder(&context) {}
 			circt::firrtl::CircuitOp MlirGen(const jlm::lambda::node *lamdaNode);
 			void WriteModuleToFile(const circt::firrtl::FModuleOp fModuleOp, const jive::node *node);
-			void WriteCircuitToFile(const circt::firrtl::CircuitOp circuit, std::string name);
+			std::string WriteCircuitToFile(const circt::firrtl::CircuitOp circuit, std::string name);
```

Comment:
Could we maybe rename this function to something else as it does not write anything to a file any longer?

Reply:
This function is used in other places, so I've created a new called toString() instead.